.env.example

```
@@ -0,0 +1,2 @@
+RELATION_DATABASE_USERNAME=root
+RELATION_DATABASE_PASSWORD=qwer1234!@
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)